### DISCUSSION TITLE
Multiple servers?

### DISCUSSION THREAD

**Discussion by SexyLexx** [apohl79/audiogridder#497] [Q&A]
First of all thank you for this great plugin. Awesome work you guys are doing. :-)

I have a question. I'm already running a remote server. If I wanna add another server, can I run my servers (with different ID's) through a switch and the one lan cable to my Slave/daw machine from the switch or does every server need a hardware input to the slave/daw machine?

**Comment by apohl79** [comment]
Yes you can. Also if the servers are running on different computers, you don't need different IDs.

**Reply by SexyLexx** [reply]
Awesome!!.. Thanks for the quick answer :-)

**Reply by JDL198** [reply]
except it doesnt - my AG plugin wil only talk to oine external server not 2 or more - help?
